contributors/devel/sig-testing/testing.md

```diff
@@ -48,17 +48,6 @@ passing, so it is often a good idea to make sure the e2e tests work as well.
 cd kubernetes
 make test  # Run all unit tests.
 ```
-If you have `GOPATH` set up correctly, you can
-also just use `go test` directly.
-
-```sh
-cd kubernetes
-go test ./... # Run all unit tests
```

Comment:
yup, wrong if we're in the base directory. It will try (and fail) to run the integration and e2e tests.